### PR TITLE
[ES] added content/es/container-orchestration.md

### LINKED ISSUE #1762: [Spanish] localize 'Container Orchestration'
Localize the `content/en/container-orchestration.md` document to Spanish

### PR DESCRIPTION
**Pull request by marianogg9**
### Describe your changes
- added `content/es/container-orchestration.md`.

### Related issue number or link (ex: `resolves #issue-number`)
- resolves #1762 

### Checklist before opening this PR (put `x` in the checkboxes)
- [X] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [X] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/es/container-orchestration.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+title: Orquestación de Contenedores
+status: Completed
+category: Concepto
+---
+
+## ¿Qué es?
+
+La Orquestación de [Contenedores](/es/container/) refiere al manejo y automatización del ciclo de vida de una aplicación contenedorizadas en ambientes dinámicos.
+Es ejecutada a través de un orquestador de containers (por lo general, [Kubernetes](/es/kubernetes)),
+el cual ofrece despliegues, autoescalado, auto reparación y monitoreo.
```
Comment (electrocucaracha) [inline content/es/container-orchestration.md:11]:
```suggestion
Es ejecutada a través de un orquestador de containers (por lo general, [Kubernetes](/es/kubernetes)), el cual ofrece despliegues, autoescalado, auto reparación y monitoreo.
```

File: content/es/container-orchestration.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+El manejo de [microservicios](/es/microservices), la seguridad, y la comunicación de red a escala - y los [sistemas distribuidos](/es/distributed-systems) en general - es difícil, sino imposible, de hacer manualmente.
+La Orquestación de Contenedores permite a los usuarios automatizar todas estas tareas operacionales.
+
+## ¿Cómo ayuda?
+
+Las herramientas de Orquestación de Contenedores facilitan al usuario a definir el estado deseado de un sistema.
+En principio, el usuario declara cómo debería ser (por ejemplo la cantidad de contenedores, asignación de recursos, etc).
+Luego, la herramienta de Orquestación automáticamente monitoreará la infraestructura y aplicará las correcciones necesarias si el estado actual del sistema se desviase del estado definido (por ejemplo lanzar un nuevo container si uno fallara).
+Esta automatización simplifica muchas de las tareas operativas de los equipos de ingeniería, que de otra manera serían por lo general complejas y ejecutadas manualmente,
+incluyendo el aprovisionamiento, despliegue, escalado (expansión y contracción), red, balanceo de carga y otras actividades.
```
Comment (electrocucaracha) [inline content/es/container-orchestration.md:26]:
```suggestion
Esta automatización simplifica muchas de las tareas operativas de los equipos de ingeniería, que de otra manera serían por lo general complejas y ejecutadas manualmente, incluyendo el aprovisionamiento, despliegue, escalado (expansión y contracción), red, balanceo de carga y otras actividades.
```

File: content/es/container-orchestration.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+---
+title: Orquestación de Contenedores
+status: Completed
+category: Concepto
+---
+
+## ¿Qué es?
+
+La Orquestación de [Contenedores](/es/container/) refiere al manejo y automatización del ciclo de vida de una aplicación contenedorizadas en ambientes dinámicos.
+Es ejecutada a través de un orquestador de containers (por lo general, [Kubernetes](/es/kubernetes)),
```
Comment (electrocucaracha) [inline content/es/container-orchestration.md:10]:
```suggestion
Es ejecutada a través de un orquestador de contenedores (por lo general, [Kubernetes](/es/kubernetes)),
```

File: content/es/container-orchestration.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+el cual ofrece despliegues, autoescalado, auto reparación y monitoreo.
+La Orquestación es una metáfora:
+Al igual que un director de orquesta, la herramienta de Orquestación dirige a los contenedores asegurándose que cada contenedor (como un músico) haga lo que debe.
```
Comment (electrocucaracha) [inline content/es/container-orchestration.md:13]:
```suggestion
Al igual que un director de orquesta, la herramienta de orquestación dirige a los contenedores asegurándose que cada contenedor (como un músico) haga lo que debe hacer.
```

File: content/es/container-orchestration.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+## Problema que aborda
+
+El manejo de [microservicios](/es/microservices), la seguridad, y la comunicación de red a escala - y los [sistemas distribuidos](/es/distributed-systems) en general - es difícil, sino imposible, de hacer manualmente.
```
Comment (electrocucaracha) [inline content/es/container-orchestration.md:17]:
```suggestion
El manejo de [microservicios](/es/microservices), la seguridad, y la comunicación de red a escala - y los [sistemas distribuidos](/es/distributed-systems) en general - es muy difícil, casi imposible, de hacer manualmente.
```

File: content/es/container-orchestration.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+La Orquestación de Contenedores permite a los usuarios automatizar todas estas tareas operacionales.
+
+## ¿Cómo ayuda?
+
+Las herramientas de Orquestación de Contenedores facilitan al usuario a definir el estado deseado de un sistema.
```
Comment (electrocucaracha) [inline content/es/container-orchestration.md:22]:
```suggestion
Las herramientas de orquestación de contenedores facilitan al usuario a definir el estado deseado de un sistema.
```

File: content/es/container-orchestration.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+En principio, el usuario declara cómo debería ser (por ejemplo la cantidad de contenedores, asignación de recursos, etc).
+Luego, la herramienta de Orquestación automáticamente monitoreará la infraestructura y aplicará las correcciones necesarias si el estado actual del sistema se desviase del estado definido (por ejemplo lanzar un nuevo container si uno fallara).
```
Comment (electrocucaracha) [inline content/es/container-orchestration.md:24]:
```suggestion
Luego, la herramienta de orquestación automáticamente monitoreará la infraestructura y aplicará las correcciones necesarias si el estado actual del sistema se desviase del estado definido (por ejemplo lanzar un nuevo container si uno fallara).
```

**Review by electrocucaracha — CHANGES_REQUESTED** [review]
Muchas gracias @marianogg9, en general el documento está muy bien, solo dejo algunas sugerencias mínimas que puede mejorar la lectura.

File: content/es/container-orchestration.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+## ¿Qué es?
+
+La orquestación de [contenedores](/es/container/) se refiere al manejo y automatización del ciclo de vida de una aplicación contenedorizada en ambientes dinámicos.
+Es ejecutada a través de un orquestador de contenedores (por lo general, [Kubernetes](/es/kubernetes)), el cual ofrece despliegues, autoescalado, auto reparación y monitoreo.
+La orquestación es metafórica:
+Al igual que un director de orquesta, la herramienta de orquestación dirige a los contenedores asegurándose que cada contenedor (como un músico) haga lo que debe hacer.
+
+## Problema que aborda
+
+El manejo de [microservicios](/es/microservices), la seguridad, y la comunicación de red a escala - y los [sistemas distribuidos](/es/distributed-systems) en general - es muy difícil, casi imposible, de hacer manualmente.
```
Comment (electrocucaracha) [inline content/es/container-orchestration.md:16]:
```suggestion
El manejo de [microservicios](/es/microservices), la seguridad, y la comunicación de red a escala - y los [sistemas distribuidos](/es/distributed-systems) en general - es muy difícil, casi imposible de hacerse manualmente.
```

**Review by electrocucaracha — APPROVED** [review]

**Review by krol3 — APPROVED** [review]
lgtm

**Pull request merged**